factcast-snapshotcache-redisson/src/main/java/org/factcast/core/snap/redisson/RedissonSnapshotCache.java

```diff
@@ -49,12 +51,16 @@ public RedissonSnapshotCache(@NonNull RedissonClient redisson, int retentionTime
   @Override
   public @NonNull Optional<Snapshot> getSnapshot(@NonNull SnapshotId id) {
     String key = createKeyFor(id);
-    RBucket<Snapshot> bucket = redisson.getBucket(key);
+    // From redisson-spring-boot-starter 3.19.0 onwards the default codec is Kryo5.
+    // Since it also uses Java17 we have to stick with 3.18.1 and enforce the old default codec
```

Comment:
the java17 thesis is probably not true.

Reply:
I tried but also failed to reproduce it.